packages/devextreme/testing/tests/DevExpress.ui.widgets/chatParts/messageList.tests.js

```diff
@@ -60,6 +65,61 @@ QUnit.module('MessageList', moduleConfig, () => {
         QUnit.test('scrollable should be rendered inside root element', function(assert) {
             assert.ok(Scrollable.getInstance(this.$element.children().first()) instanceof Scrollable);
         });
+
+        QUnit.test('Message Group should be rendered in the scrollable content', function(assert) {
+            const newMessage = {
+                author: { id: MOCK_CURRENT_USER_ID },
+                timestamp: NOW,
+                text: 'NEW MESSAGE',
+            };
+
+            this.reinit({ items: [newMessage] });
```

Comment:
Seem, that scenario in the next test is the same.

Reply:
In the next test we check at runtime